### PR TITLE
Story/quad 108

### PR DESCRIPTION
**Pull request by ajaymodi**

### CODE REVIEW

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by ajaymodi** [comment]
Entire files are coming in conflicts, does only indentation change for those files or anything else as well?

Does it mean I am using wrong indentation?

**Comment by jdamerow** [comment]
I'm sorry, I did some refactoring and also reformatted the files that I changed. I was hoping that nobody would be actually working on those files, and only realized in the end that you made changes to at least one of them. Another thing, that I was going to mention in class on Friday is that if you are on a Windows machine you should set Git's autocrlf to true (see core.autocrlf here: https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration). But you're on Linux, right? So unless you have autocrlf set to input, that shouldn't be an issue.

**Comment by ajaymodi** [comment]
Hmm... I tried to resolve conflicts based on that, let me know if you want me to roll back any changes.

Yes, I am using linux machine and I do not see anything related to autocrlf set in my git config so in that case I will be using default option.

File: Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStatsTest.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        List<IConceptStats> conceptStatsList = projectStatsUnderTest
+                .getConceptCount(mockedNetworkList);
+        assertTrue(conceptStatsList.containsAll(mockedConceptStatsList)
+                && mockedConceptStatsList.containsAll(conceptStatsList));
+
+    }
+
+    @Test
+    public void getSortedConceptCountTest() throws QuadrigaStorageException {
+        assertEquals("conceptCount", mockedConceptStatsList,
```
Comment (jdamerow) [inline Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStatsTest.java]: Why do you use a "conceptCount" message here? Either don't put anything, or make it a real message.

**Comment by jdamerow** [comment]
ok to test

File: Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStatsTest.java (hunk 1 above)
Comment (jdamerow) [inline Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStatsTest.java]: never mind. I'll fix it real quick.

Comment (ajaymodi) [inline Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/ProjectStatsTest.java]: Thanks.

**Pull request merged**